project/components/resource_request/schemas.py

```diff
@@ -46,10 +50,17 @@ class ResourceRequestResponseSchema(ResourceRequestSchema):
     """Default schema for single resource request in response."""
 
     id: UUID
+    project: Union[ProjectResponseSchema, Dict[str, str]]
```

Comment:
Then use schema here.
```suggestion
    project: EmbeddedProjectSchema
```